frontend/packages/ceph-storage-plugin/src/components/modals/storage-class-dropdown.tsx

```diff
@@ -1,10 +1,11 @@
 import * as React from 'react';
 import * as _ from 'lodash';
 import { Firehose } from '@console/internal/components/utils';
+import { InfrastructureModel } from '@console/internal/models';
+import { K8sResourceKind, StorageClassResourceKind, k8sGet } from '@console/internal/module/k8s';
```

Comment:
Alpha imports

Reply:
Already alpha. capital `K` and small `k` diff.

Reply:
ohk